sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -730,6 +730,935 @@ public async Task CreateAndGetSqlServerDataFeedWithOptionalMembers()
             Assert.That(createdDataSource.Query, Is.EqualTo("query"));
         }
 
+        [RecordedTest]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/17726")]
+        public async Task UpdateAzureApplicationInsightsDataFeedWithMinimumSetupAndGetInstance()
+        {
+            MetricsAdvisorAdministrationClient adminClient = GetMetricsAdvisorAdministrationClient();
+
+            var dataFeedName = Recording.GenerateAlphaNumericId("dataFeed");
+            var dataSource = new AzureApplicationInsightsDataFeedSource("appId", "key", "cloud", "query");
```

Comment:
nit: extract the strings out to variables to be used in the asserts below.

Reply:
Done!